packages/text-field/index.tsx

```diff
@@ -329,7 +329,7 @@ class TextField<T extends {}> extends React.Component<TextFieldProps<T>, TextFie
         key='text-field-container'
       >
         {leadingIcon ? this.renderIcon(leadingIcon, onLeadingIconSelect) : null}
-        {foundation ? this.renderInput() : null}
+        {this.renderInput()}
```

Comment:
This isn't needed anymore?

Reply:
this fix is because of this line, actually